src/search_datastructures.hpp

```diff
@@ -339,10 +341,11 @@ class GlobalDataHolder
     static constexpr AlphabetEnum  c_redAlph       = c_redAlph_;
     static constexpr AlphabetEnum  c_origQryAlph   = c_origQryAlph_;
 
-    using TOrigQryAlph   = _alphabetEnumToType<c_origQryAlph>;
-    using TOrigSbjAlph   = _alphabetEnumToType<c_origSbjAlph>;
-    using TTransAlph     = _alphabetEnumToType<c_transAlph>;
-    using TRedAlph       = _alphabetEnumToType<c_redAlph>;
+    using TOrigQryAlph   = _alphabetEnumToType<c_origQryAlph, true>;
+    using TOrigSbjAlph   = _alphabetEnumToType<c_origSbjAlph, true>;
+    using TTransAlph     = _alphabetEnumToType<c_transAlph, true>;
```

Comment:
Why do these get an extra parameter, it is always just the reduced alphabet that can change, or not?